# Upstream: http://bugzilla.gnome.org/show_bug.cgi?id=539095
# Description: rename functions due to the upstream offscreen rendering changes
#
Index: gtk+2.0-2.13.3/gdk/directfb/gdkwindow-directfb.c
===================================================================
--- gtk+2.0-2.13.3.orig/gdk/directfb/gdkwindow-directfb.c	2008-06-18 17:06:14.000000000 +0200
+++ gtk+2.0-2.13.3/gdk/directfb/gdkwindow-directfb.c	2008-06-18 17:06:30.000000000 +0200
@@ -646,7 +646,7 @@
 }
 
 GdkWindow *
-gdk_window_new (GdkWindow     *parent,
+_gdk_window_new (GdkWindow     *parent,
                 GdkWindowAttr *attributes,
                 gint           attributes_mask)
 {
@@ -1511,7 +1511,7 @@
 }
 
 void
-gdk_window_reparent (GdkWindow *window,
+_gdk_window_reparent (GdkWindow *window,
                      GdkWindow *new_parent,
                      gint       x,
                      gint       y)
